CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
-### 🔄 Changed
+## StreamChat
+### 🐞 Fixed
+- Reduce channel list updates when syncing by reording syncing sequence and skipping channel based events sync (if already in sync) [#3450](https://github.com/GetStream/stream-chat-swift/pull/3450)
```

Comment:
- typo: reording
- the entry itself is a bit long and not clear enough, maybe we can simplify it?

Reply:
Thank you, sure